builder/docker/ecr_login.go

```diff
@@ -38,7 +39,10 @@ func (c *AwsAccessConfig) config(region string) (*aws.Config, error) {
 			SessionToken:    c.Token,
 		}},
 		&credentials.EnvProvider{},
-		&credentials.SharedCredentialsProvider{Filename: "", Profile: ""},
+		&credentials.SharedCredentialsProvider{
```

Comment:
this seems like it would be fine, but it would be nice to have some agreement with the aws builder's access config. https://github.com/hashicorp/packer/blob/8585e0ebf5a9a83afe32064eeb16d29210d6fd1a/builder/amazon/common/access_config.go#L40-L80

I wonder if we shouldn't just use that instead, like we do in the amazon-import post-processor.

I won't block this PR on that, but if you wanted to take a swing at it, I think it would be better than continuing with two methods

Reply:
I'm all in favor of converging on one aws access config. I'll look into it and update here.